scheduler/src/cook/kubernetes/controller.clj

```diff
@@ -128,9 +137,17 @@
     {:cook-expected-state :cook-expected-state/completed}))
 
 (defn launch-pod
-  [{:keys [api-client] :as compute-cluster} cook-expected-state-dict pod-name]
-  (if (api/launch-pod api-client cook-expected-state-dict pod-name)
-    cook-expected-state-dict
+  [{:keys [api-client name] :as compute-cluster} cook-expected-state-dict pod-name]
+  (if (api/launch-pod api-client name cook-expected-state-dict pod-name)
+    ; Save a timer so that we can track the actual launch time (histogram) for k8s to come back to waiting and running.
+    ; Note, this is intended for the happy path, identify regressions in autoscaling and other startup.
+    ; This is intended to measure the time after the API call returns. (The API call is already measured)
+    ;
+    ; k8s-response-time-until-waiting helps us characterize watch latency, and if watch latency spikees or goes wrong.
+    ; k8s-response-time-until-running can help us understand pod startup time (and auto-scaling time)
```

Comment:
I don't think this comment is needed; we don't need to justify the metrics, and their names explain what they're measuring.

Reply:
Explaining the scope is useful. This isn't supposed to measure unhappy paths. (e.g., a task that fails to launch), and I want to be clear about the intentions, so people don't wonder if there's a bug hidden here. This also is intended to not include API call time (so if you want to get the true time to launch, you'd need to add the API call time back)

Reply:
Only one of the sentences in the comment adds any information that I don't easily get from the code itself:

> k8s-response-time-until-waiting helps us characterize watch latency

The rest of that comment is superfluous and distracting in my opinion.

Reply:
Discussed offline, we're going to whittle down this comment to the following:

> These metrics only measure the happy paths to avoid wide variations from error rates changing.